lessons/specifics/mnesia.md

```diff
@@ -113,6 +113,13 @@ We define the columns using the atoms `:id`, `:name`, and `:job`. When we execut
  - `{:atomic, :ok}` if the function executes successfully
  - `{:aborted, Reason}` if the function failed
 
+In particular, if the table already exists, the reason will be of the form `{:already_exists, table}` so if we try to create this table a second time, we will get:
+
+```shell
```

Comment:
In other sections for highlighting we have used the `elixir` tag instead of `shell`, but I dont know if this is an official convention (but in my PRs it has been the style I've used and quickly referencing some other parts it seems to be the norm).

CC @doomspork cuz I'm not sure tbh